app/jobs/bulkrax/import_work_collection_job.rb

```diff
@@ -19,5 +20,26 @@ def perform(*args)
       end
     end
     # rubocop:enable Rails/SkipsModelValidations
+
+    private
+
+      def add_user_to_permission_template!(entry)
+        user                = ::User.find(entry.importerexporter.user_id)
+        collection          = entry.factory.find
+        permission_template = Hyrax::PermissionTemplate.find_by(source_id: collection.id)
+
+        if permission_template.present?
+          Hyrax::PermissionTemplateAccess.create!(
+            permission_template_id: permission_template.id,
+            agent_id: user.user_key,
+            agent_type: 'user',
+            access: 'manage'
+          )
+        else
+          Hyrax::PermissionTemplate.create!(source_id: collection.id, manage_users: [user])
```

Comment:
I was testing collection creation via a CsvImporter with this branch, and I think `manage_users` is not a valid attribute here?

Reply:
@dlpierce I think you're right, I can't remember exactly where I got `manage_users` from. 

Looking at this again, I think it would make more sense to do something like: 
```ruby
...
permission_template = Hyrax::PermissionTemplate.find_by(source_id: collection.id)

Hyrax::PermissionTemplate.create!(source_id: collection.id) unless permission_template

Hyrax::PermissionTemplateAccess.create!(
  permission_template_id: permission_template.id,
  agent_id: user.user_key,
  agent_type: 'user',
  access: 'manage'
)
...
```

What do you think?